cmd/go2proto/testdata/go2proto.to.go

```diff
@@ -56,6 +57,14 @@ func orZero[T any](v *T) T {
 	return *v
 }
 
+func optionalOrZero[T any](v optional.Option[T]) *T {
+	if v.Ok() {
+		r := v.MustGet()
+		return &r
+	}
+	return nil
+}
+
```

Comment:
You don't need this function at all, just use `v.Ptr()` and it will do the same thing.